### DISCUSSION TITLE
Installation issue on dedicated laptop

### DISCUSSION THREAD

**Discussion by neilmadhava** [Security-Onion-Solutions/securityonion#8763] [General] [answered]
Hello,
I'm trying to install Security Onion 2.3.160-20220829 on Dell Latitude 5410. I used balena etcher to create a bootable USB. I don't have internet access and am trying to do an airgap installation. I have verified the ISO file using the VERIFY_ISO.md doc. I get the following error when I try to install the OS. I've tried installing from the USB on multiple Dell Latitude 5410 and 5420 modules with the same result. The USB worked on Dell 7400 model and an old Lenovo Ideapad 15IKB (I did not go through with the installation on these laptops but the error in the image below did not appear and I could get to the "Do you wish to continue" page.) 

![image](https://user-images.githubusercontent.com/30976668/190984454-b436bc41-2064-4be0-a61d-0e91cb58c9f4.png)

**Error**:
Kernel BUG at include/linux/dma-attrs.h:53!
invalid opcode: 0000 [#1] SMP
Kernel Panic - not syncing: Fatal exception

**System Information**:
Memory Installed = 16 GB
Memory Technology = DDR4 SDRAM
Processor Type = Intel(R) Core(TM) i5-10310U CPU @1.70GHz
Core Count = 4

I've already tried booting from a USB with just CentOS 7 and it works fine. I can install CentOS and Ubuntu on the laptop. But since I do not have access to internet in my test environment, I cannot do a network install of Security Onion.

Any help would be appreciated.

**Comment by dougburks** [comment, marked as answer]
Since you don't have Internet access, your best bet would be to find different hardware that works with our ISO image.

**Reply by neilmadhava** [reply]
I tested the deployment on a Dell Optiplex 3050 today (didn't go through with the installation because I needed to get more RAM in there) and I didn't get the kernel panic error. For now, it looks like the error is unique to Dell Latitude 5410/5420 models.
